test/XRP/wallet-factory.test.ts

```diff
@@ -2,11 +2,12 @@ import { assert } from 'chai'
 
 import WalletFactory from '../../src/XRP/wallet-factory'
 import 'mocha'
+import XRPLNetwork from '../../src/XRP/xrpl-network'
```

Comment:
should the casing here be `XrplNetwork` ?

Reply:
Old habits die hard.